### ISSUE TITLE
Changing MathJax output format

### ISSUE THREAD

**Issue by blubbsy** [hakimel/reveal.js#1612]
Hi,

i'm preparing a presentation with reveal.js and mathjax. I'm trying to use the same font in mathjax as in reveal.js but iam not able to set the config right for mathjax and it always gives me the output in the standard font (i think its called STIX). What i want is the font used with the PreviewHTML renderer, but the only way i found to use it was to delete or to rename the output folders of mathjax, which is a very unnice way to solve the problem. Does someone know how i can change in a nice and correct way the input and output format? 

The mathjax documentation says this (http://docs.mathjax.org/en/latest/output.html):

`jax: ["input/TeX","output/HTML-CSS"]`

so basically, as far as i see it, i have to set jax to, but i dont know how:

`jax: ["input/TeX","output/PreviewHTML"]`

Greetings,

Blubbsy

**Comment by blubbsy** [comment]
update:

i started to work in math.js and found out that i can choose the renderer with `MathJax.Hub.setRenderer("PreviewHTML");`, but then i have the problem that i receive a "Math processing error".

**Comment by xumiao** [comment]
why not change the font of html-css? according to the doc, the default preferred font seems to be Tex.
http://docs.mathjax.org/en/latest/options/HTML-CSS.html#configure-html-css
